inst/tests/tests.Rraw

```diff
@@ -5994,7 +5996,9 @@ test(1444.4, fread(paste(paste("Hz.BB.GHG.", 1:10, sep = ""), collapse="\n"), se
              data.table(V1="Hz",V2="BB",V3="GHG",V4=1:10))
 
 # doubled quote inside a quoted field followed by an embedded newline
+if (.Platform$OS.type == "unix") {
 test(1445, fread("doublequote_newline.csv")[7:10], data.table(A=c(1L,1L,2L,1L), B=c("a","embedded \"\"field\"\"\nwith some embedded new\nlines as well","not this one","a")))
+}
```

Comment:
This is passing successfully on AppVeyor Windows tests and on CRAN Windows tests.  Something else must be going wrong on your Windows machine, to fix separately.  Important to keep the test running on Windows.

Reply:
Would you prefer I raise this an issue and revert this change, or relax the test to permit e.g. `embedded ""field""\r\nwith some embedded new\r\nlines as well`  as well as `embedded ""field""\nwith some embedded new\nlines as well`?

Reply:
Yes : new issue please and revert the change.  Why does it pass on AppVeyor and CRAN WIndows but it doesn't pass on your Windows?

Reply:
Ahh, I think it's because when I clone the repo, `\n` is replaced with `\r\n` by **git**. My bad. 

```r
fread("https://raw.githubusercontent.com/Rdatatable/data.table/master/inst/tests/doublequote_newline.csv")[["B"]][8]
#> [1] "embedded \"\"field\"\"\nwith some embedded new\nlines as well"

fread("inst/tests/doublequote_newline.csv")[["B"]][8]
#> [1] "embedded \"\"field\"\"\r\nwith some embedded new\r\nlines as well"
```